### PR TITLE
docs(persist): Reinforce document

### PR DESCRIPTION
**Pull request by happyhj**
## Issue
#297 

## Details
Update on descriptions

## Preferred reviewers
@naver/egjs-dev

### CODE REVIEW

File: src/plugin/persist.js [hunk 1]
```diff
@@ -169,12 +169,12 @@ eg.module("persist", ["jQuery", eg, window, document], function($, ns, global, d
 	}
 
 	/**
-	* Save current state.
-	* @ko 인자로 넘긴 현재 상태정보를 저장한다.
+	* Serialize given state as JSON and save in global namespace.
+	* @ko 받은 상태를 글로벌 네임스페이스에 JSON 으로 직렬화하여 저장한다
```
Comment (netil) [inline src/plugin/persist.js]: I think '받은 상태' sounds weird.
'전달된 상태값을' or simply '상태값을'

Comment (jongmoon) [inline src/plugin/persist.js]: @happyhj I think `serialze`  is not needed in a document.

File: src/plugin/persist.js [hunk 2]
```diff
@@ -169,8 +169,8 @@ eg.module("persist", ["jQuery", eg, window, document], function($, ns, global, d
 	}
 
 	/**
-	* Serialize given state as JSON and save in global namespace.
-	* @ko 받은 상태를 글로벌 네임스페이스에 JSON 으로 직렬화하여 저장한다
+	* Save state as JSON in global namespace.
+	* @ko 상태값을 글로벌 네임스페이스에 JSON 으로 저장한다
```
Comment (jongmoon) [inline src/plugin/persist.js]: @happyhj I don't know what is global namespace exactly.

**Comment by netil** [comment]
LGTM

**Comment by jongmoon** [comment]
LGTM

**Pull request merged**